adafruit_emc2101/emc2101_lut.py

```diff
@@ -34,125 +35,33 @@
 and PWM frequency control to the base feature set.
 """
 
-from micropython import const
-from adafruit_register.i2c_struct_array import StructArray
 from adafruit_register.i2c_struct import UnaryStruct
-from adafruit_register.i2c_bit import RWBit
-from . import EMC2101, MAX_LUT_SPEED, MAX_LUT_TEMP
+
+import emc2101_regs
+from emc2101_fanspeed import FanSpeedLUT
+from emc2101_ext import EMC2101_EXT
```

Comment:
I believe these should be relative imports:

```python
from adafruit_emc2101 import emc2101_regs
...
```

Reply:
This is fixed in the latest commit. Changed these imports to use adafruit_2101 and removed the insert path inside of `conf.py` bandaide.